examples/app-mvvm/app.json

```diff
@@ -0,0 +1,335 @@
+     */
+    "framework": "ext",
+
+    /**
+     * The name of the theme for this application.
+     */
+    
+    "theme": "ext-theme-crisp",
+    
+    /**
+     * The list of required packages (with optional versions; default is "latest").
+     *
+     * For example,
+     *
+     *      "requires": [
+     *          "sencha-charts"
+     *      ]
+     */
+    "requires": [
+        "GeoExt",
+        "sencha-charts"
+    ],
+
+    /**
+     * Sass configuration properties.
+     */
+    "sass": {
+        /**
+         * The root namespace to use when mapping *.scss files to classes in the
+         * sass/src and sass/var directories. For example, "CF.view.Foo" would
+         * map to "sass/src/view/Foo.scss". If we changed this to "CF.view" then
+         * it would map to "sass/src/Foo.scss". To style classes outside the app's
+         * root namespace, change this to "". Doing so would change the mapping of
+         * "CF.view.Foo" to "sass/src/CF/view/Foo.scss".
+         */
+        "namespace": "CF"
+    },
+
+    /**
+     * List of all JavaScript assets in the right execution order.
+     *
+     * Each item is an object with the following format:
+     *
+     *      {
+     *          // Path to file. If the file is local this must be a relative path from
+     *          // this app.json file.
+     *          //
+     *          "path": "path/to/script.js",   // REQUIRED
+     *
+     *          // Set to true on one file to indicate that it should become the container
+     *          // for the concatenated classes.
+     *          //
+     *          "bundle": false,    // OPTIONAL
+     *
+     *          // Set to true to include this file in the concatenated classes.
+     *          //
+     *          "includeInBundle": false,  // OPTIONAL
+     *
+     *          // Specify as true if this file is remote and should not be copied into the
+     *          // build folder. Defaults to false for a local file which will be copied.
+     *          //
+     *          "remote": false,    // OPTIONAL
+     *
+     *          // If not specified, this file will only be loaded once, and cached inside
+     *          // localStorage until this value is changed. You can specify:
+     *          //
+     *          //   - "delta" to enable over-the-air delta update for this file
+     *          //   - "full" means full update will be made when this file changes
+     *          //
+     *          "update": "",        // OPTIONAL
+     *
+     *          // A value of true indicates that is a development mode only dependency.
+     *          // These files will not be copied into the build directory or referenced
+     *          // in the generate app.json manifest for the micro loader.
+     *          //
+     *          "bootstrap": false   // OPTIONAL
+     *      }
+     *
+     */
+    "js": [
+        {
+            "path": "http://cdn.sencha.com/ext/gpl/5.1.0/build/ext-all-rtl-debug.js"
```

Comment:
Is this a "right-to-left" build of Ext? I think this can be omitted for development all together an the microloader will pick up your local Ext files.